mistralrs-server/src/main.rs

```diff
@@ -109,7 +109,10 @@ impl futures::Stream for Streamer {
         }
         match self.rx.try_recv() {
             Ok(resp) => match resp {
-                Response::Error(e) => {
```

Comment:
It looks like we ignore Response::ModelError, is there a reason why we do not handle it? Additionally, I think we should mark the streamer as done when an error comes through to prevent a hang, I wish we could signal that we got an error.

Reply:
No reason at all. The `match` used a wildcard and I did not see that I should add another arm.

> I think we should mark the streamer as done when an error comes through to prevent a hang, I wish we could signal that we got an error.

This is my first time dealing with SSE, I also don't know what's the best way to deal with it...

Reply:
I checked the Event docs, and there doesn't seem to be a way to signal an error. The only option is to set it in the comment, but I think that it would be hard to find there.